activities/staff-management/travel.md

```diff
@@ -20,7 +20,7 @@ Book transportation according to our [remuneration policy](../../organization/re
 Tips:
 
 * Make sure you have good connections as well as leaving time and space to settle into a hotel.
-* Travelling inside business hours might help you make sure you don't have to go to bed without dinner before representing the Foundation for Public Code the next morning.
-* Make sure your accomodation has a stable internet connection and access to necessary facilities like an ironing board.
-* When attending an event, it might pay to have accomodation near the event in order to maximize participation.
+* Traveling inside business hours might help you make sure you don't have to go to bed without dinner before representing the Foundation for Public Code the next morning.
+* Make sure your accommodation has a stable internet connection and access to necessary facilities like an ironing board.
+* When attending an event, it might pay to have accommodation near the event in order to maximize participation. If several staff members travel to the same event, unless there are specific reasons not to, stay at the same place.
```

Comment:
Perhaps add that booking together can also be worthwhile :)

Reply:
It might, but it can also be quite a hefty expense for one person booking hotel for five persons and three days.

Reply:
This can also be booked by our 'procurement' departmens, e.g. those of us with access to money. However Hotels are often poor with booking for others.